db/migrate/20190607110030_add_indexes_for_analytics.rb

```diff
@@ -0,0 +1,16 @@
+class AddIndexesForAnalytics < ActiveRecord::Migration[5.2]
+  # transactions are disabled to allow PostgreSQL to add indexes concurrently
+  disable_ddl_transaction!
+
+  def change
+    # concurrent creation avoids downtime during production deploy because
+    # indexes get created in the background
```

Comment:
This is really great to know!

Reply:
Yeah, it's a neat feature.

With 9.6 they went beyond that introducing basic parallelism features (some queries are transparently run in parallel by the DB) and expanded them in versions 10 and 11 (more types of parallel queries but also parallel joins and parallel creation of other types of indexes, you can also set the number of workers and other things). More details in my changelog issue https://github.com/thepracticaldev/dev.to/issues/2801